test/functional/rpc_scanblockfilters.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test the scantxoutset rpc call."""
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+class ScanblockfiltersTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 1
+        self.extra_args = [["-blockfilterindex=1"]]
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+
+    def run_test(self):
+        addr_1 = self.nodes[0].getnewaddress()
+        self.nodes[0].sendtoaddress(addr_1, 1.0)
+        assert_equal(self.nodes[0].scanblockfilters(["addr("+addr_1+")"]), [])
+        blockhash = self.nodes[0].generate(1)[0]
+        assert(blockhash in self.nodes[0].scanblockfilters(["addr("+addr_1+")"]))
+        blockhash_new = self.nodes[0].generate(1)[0]
+        self.log.info("Test scanblockfilters startheight...")
+        assert(blockhash not in self.nodes[0].scanblockfilters(["addr("+addr_1+")"], self.nodes[0].getblockheader(blockhash_new)['height']))
```

Comment:
This is a bit hard to read, maybe start with `start_height = self.nodes[0].getblockheader(blockhash_new)['height']`